app/javascript/components/taggingWrapper.jsx

```diff
@@ -4,10 +4,15 @@ import PropTypes from 'prop-types';
 import { Loading } from 'carbon-components-react';
 import { TaggingWithButtonsConnected, TaggingConnected, taggingApp } from '../tagging';
 
+const SelectedTags = (state, tag) => {
```

Comment:
minor, but should this be lower case?

```suggestion
const selectedTags = (state, tag) => {
```

Reply:
Fixed it.